srcpkgs/nfs4-acl-tools/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'nfs4-acl-tools'
+pkgname=nfs4-acl-tools
+version=0.3.5
+revision=1
+build_style=configure
+configure_args="--prefix=/usr"
+hostmakedepends="automake autoconf"
+makedepends="libtool attr-devel"
+short_desc="Client tools for manipulating NFSv4 ACLs directly"
+maintainer="Giuseppe Fierro <gspe@ae-design.ws>"
+license=" BSD-3-Clause, LGPL-2.1, GPL-2.0-or-later"
```

Comment:
LGPL-2.1 isn't a SPDX short-identifier

Reply:
Thanks, I forgot to change this.
I have a question, how do you see if a license is *-or-later or *-only version?

Reply:
You have to take a look at the source files e.g. http://git.linux-nfs.org/?p=bfields/nfs4-acl-tools.git;a=blob;f=include/buildrules;h=6dfacde501cc88578811781ff94da629671662db;hb=HEAD is GPL-2.0-only. I cannot find any file that is LGPL licensed.

Reply:
@Johnnynator thanks.
Other licenses are mentioned in COPYING file:
http://git.linux-nfs.org/?p=bfields/nfs4-acl-tools.git;a=blob;f=COPYING;h=3b4e71a9d724c3500c41027c5ba2877de98baf76;hb=f83c544cc6d777de8dbdde2a0be76cf5895aae25

Reply:
@gspe I know. The LGPL part is probably just copied from acl. The only important part is the BSD License since only the build system is GPL which is not in the final package. (Also acl is LGPL-2.1-or-later)